frontend/src/js/components/Groups/Item.react.js

```diff
@@ -144,8 +144,7 @@ function Item(props) {
               <CardFeatureLabel>Rollout Policy</CardFeatureLabel>
               <Box p={1} mb={1}>
                 <CardLabel>
-                  Max {props.group.policy_max_updates_per_period}
-                  updates per {props.group.policy_period_interval}
+                  {`Max ${props.group.policy_max_updates_per_period || 0} updates per ${props.group.policy_period_interval || 0}`}
```

Comment:
This seems to follow what we have in `ItemExtended.react.js`, but in worst case we are going to have a message like `Max 0 updates per 0`, which does not make sense. Maybe it should be something like: 
```suggestion
                  {`Max ${props.group.policy_max_updates_per_period || 0} updates per ${props.group.policy_period_interval || '0 minutes'}`}
```

Reply:
Let's make this right then. Let's see what the real numbers mean in terms of behavior in the backend and do the changes accordingly. If 0 is possible in both places, let's change the message to what it means (`No updates allowed` / `As many updates at the same time as possible`).

Probably we shouldn't allow 0 if it means no updates, as for that we already have a main switch.